spooky-doodle/Bangazon-Website
# Complete order

# Description

Implemented complete checkout features.

Fixes #32

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Login.
Add items to cart.
Go to cart.
Press Checkout.
Pick a payment type.
Press Complete Order.
You should now see the confirmation screen, with a handy dandy confirmation number.




# Checklist:
